core/modules/flowgen.cc

```diff
@@ -200,6 +200,13 @@ pb_error_t FlowGen::ProcessArguments(const bess::pb::FlowGenArg &arg) {
     quick_rampup_ = 1;
   }
 
+  if (arg.burst() <= bess::PacketBatch::kMaxBurst) {
```

Comment:
`update()` is not thread safe, so one must pause the entire pipeline with `bess.pause_all()`